packages/rum-core/test/performance-monitoring/transaction-service.spec.js

```diff
@@ -498,12 +498,13 @@ describe('TransactionService', function () {
     const lastSpan = transaction.startSpan('last-span-name', 'last-span')
     lastSpan.end()
     longSpan.end()
-    longSpan.end += 500
+    longSpan._end += 500
```

Comment:
This test was giving a false positive. We were adding the +500 to the wrong property and we were not taking into account that `adjustTransactionTime` also truncates spans. 

The truncate point is important because it mutates the spans being used in the test. 
After adjustTransactionTime execution **transaction._end** and **longSpan_end** were equal.  That's why the assertion was still passing properly.

Removing the code that was intended to be tested was not affecting the results of the test at all.

Besides setting the +500 to the correct property I also assigned the longSpan end to a new variable (which is the one now used in the assertion) so we make sure the span truncation does not affect the results of the test anymore.